PlutoHelperAgent/AppDelegate.m

```diff
@@ -227,6 +300,34 @@ - (NSString *)getPipeData:(NSPipe *)pipe {
     NSData *d = [fp readDataToEndOfFile];
     return [[NSString alloc] initWithData:d encoding:NSUTF8StringEncoding];
 }
+   
+- (void)getVersionData {
```

Comment:
please put a comment in here explaining the purpose of this function

Reply:
Done.

Reply:
👍